examples/suite1/save.toml

```diff
@@ -10,4 +11,11 @@ lineCaptureGroup = 1  # index of regex capture group for line number, used when
 columnCaptureGroup = 2  # index of regex capture group for column number, used when `warningTextHasColumn: false`
 messageCaptureGroup = 3  # index of regex capture group for message text
 warningTextHasColumn = true
-warningTextHasLine = true
\ No newline at end of file
+warningTextHasLine = true
+
+[diff]
+exec_cmd="./ktlint -R diktat-0.4.2.jar -F"
+testFilePattern="*Test.kt"
+expectedFilePattern="*Expected.kt"
+output = "stdout" # you can also use 'inPlace' here to do fixes right into the test file (test files won't be broken or changed)
```

Comment:
With current implementation I believe they *will* be changed, what do you mean? Also, this setting should be separate from `output`, doesn't it?

Reply:
#88